asset-list.tsv

```diff
@@ -1,7 +1,7 @@
 Asset Name	Description	Filename	Game Designer Notes	Sound Designer Notes
```

Comment:
@ericbey the only fields you need to update are `Filename` and `Sound Designer Notes`